tests/functional/spec/checkbox.spec.js

```diff
@@ -39,6 +39,13 @@ describe('Checkbox with "other" option', () => {
     expect($(MandatoryCheckboxPage.error()).isDisplayed()).to.be.true;
   });
 
+  it("Given a mandatory checkbox answer, When I leave the input field empty and submit, Then an error containing the question text should be displayed.", () => {
```

Comment:
We should be clear and say that the question text is visibly hidden, as currently this implies the question text should always be displayed.